store/copr/mpp.go

```diff
@@ -268,9 +270,12 @@ func (m *mppIterator) handleDispatchReq(ctx context.Context, bo *Backoffer, req
 		return
 	}
 	if len(realResp.RetryRegions) > 0 {
-		for _, retry := range realResp.RetryRegions {
+		logutil.BgLogger().Info("TiFlash found " + strconv.Itoa(len(realResp.RetryRegions)) + " stale regions.")
+		for index, retry := range realResp.RetryRegions {
 			id := tikv.NewRegionVerID(retry.Id, retry.RegionEpoch.ConfVer, retry.RegionEpoch.Version)
-			logutil.BgLogger().Info("invalid region because tiflash detected stale region", zap.String("region id", id.String()))
+			if index < 10 || log.GetLevel() <= zap.DebugLevel {
+				logutil.BgLogger().Info("invalid region because tiflash detected stale region", zap.String("region id", id.String()))
+			}
```

Comment:
although the above tell us how many regions there are, I still suggest to add a log to tell the user(or developer) like `some regions are omitted because not in debug log mode.`

Reply:
I think its fine since line 274 tells the user about it.